### DISCUSSION TITLE
Nearby networking really unstable with a large number of devices

### DISCUSSION THREAD

**Discussion by sarbogast** [google/nearby#2452] [Q&A]
I am trying to integrate Google Nearby Connections into an Android app to create a star network because a server tablet and multiple client tablets and send messages.
The problem I have is that everything seems to work find in my development environment with 2 devices, but in the real world, with 25 devices over a couple of hours, there are a lot of disconnections, especially coming from the advertiser.
Is there any known network size limitation or other issue with this technology that doesn't make it suitable for this kind of use case?

**Comment by IgVelasco** [comment]
@bourdakos1 do you have any information on the connection capabilities?
